<ui version="4.0" >
 <class>mkEquery</class>
 <widget class="QDialog" name="mkEquery" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>48</y>
     <width>309</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Download  -  eQSL.cc</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>64</x>
     <y>84</y>
     <width>469</width>
     <height>17</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>120</y>
     <width>313</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Abfrage Queries  ( get QSL records )</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>168</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 1  All InBox records</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>196</y>
     <width>273</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 2  Confirmed Only</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>224</y>
     <width>277</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 3  UnConfirmed Only</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>256</y>
     <width>153</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 4  Rcvd Since</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>288</y>
     <width>309</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 5  All Archived records Only</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRecord" >
   <property name="geometry" >
    <rect>
     <x>88</x>
     <y>416</y>
     <width>197</width>
     <height>32</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>204</red>
         <green>222</green>
         <blue>203</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>204</red>
         <green>222</green>
         <blue>203</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>204</red>
         <green>222</green>
         <blue>203</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text" >
    <string>Download</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit" >
   <property name="geometry" >
    <rect>
     <x>348</x>
     <y>416</y>
     <width>193</width>
     <height>32</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>205</red>
         <green>204</green>
         <blue>204</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>205</red>
         <green>204</green>
         <blue>204</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>205</red>
         <green>204</green>
         <blue>204</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit" >
   <property name="geometry" >
    <rect>
     <x>252</x>
     <y>252</y>
     <width>125</width>
     <height>33</height>
    </rect>
   </property>
   <property name="palette" >
    <palette>
     <active>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>238</red>
         <green>255</green>
         <blue>230</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>238</red>
         <green>255</green>
         <blue>230</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base" >
       <brush brushstyle="SolidPattern" >
        <color alpha="255" >
         <red>244</red>
         <green>244</green>
         <blue>244</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="displayFormat" >
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_7" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>364</y>
     <width>297</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 7  Check QSL Card Status</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6" >
   <property name="geometry" >
    <rect>
     <x>84</x>
     <y>320</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string> 6  AWARD Garanting VERIFICATION</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
